lib/units/api/controllers/user.js

```diff
@@ -400,3 +401,56 @@ function getUserAccessTokens(req, res) {
       })
     })
 }
+
+function addAdbPublicKey(req, res) {
+  var data = req.swagger.params.adb.value
+  adbkit.util.parsePublicKey(data.publickey)
+    .then(function(key) {
+      return dbapi.lookupUsersByAdbKey(key.fingerprint)
+        .then(function(cursor) {
+          return cursor.toArray()
+        })
+        .then(function(users) {
+          return {
+            key: {
+              title: data.title || key.comment
+              , fingerprint: key.fingerprint
```

Comment:
Small thing but could you reduce the indentation on every line that starts with a comma? The fields should line up.